README.md

```diff
@@ -1,11 +1,97 @@
-# service-ca
-Coming soon...
+# OpenShift Service CA Operator
 
-Controller to mint and manage serving certificates for Kubernetes services.
+This operator runs the following OpenShift service-serving certificate controllers:
+* service-ca controller:
+  * Issues serving certificates based on annotations on services, and provides them via secrets. [See the current OKD documentation for usage.](https://docs.okd.io/latest/dev_guide/secrets.html#service-serving-certificate-secrets)
 
+* configmap-cabundle-injector controller:
+  * Watches for configmaps in any namespace annotated with 'service.alpha.openshift.io/inject-cabundle=true' and adds or updates a data item (key "cabundle.crt") containing the PEM-encoded CA signing bundle. Consumers of the configmap can then trust cabundle.crt in their TLS client configuration, allowing connections to services that utilize service-serving certificates.
```

Comment:
> any namespace annotated with 'service.alpha.openshift.io/inject-cabundle=true'
I wish there was a way to control this operator's scope via RBAC instead of namespace annotations 🙁 . Maybe one day.

Reply:
How are we expecting this annotation to be added to the namespace in the first place?  Normal users cannot edit a namespace.

Reply:
I think that's the point. It's a hack to give administrators control over who gets access to the CA in a dynamic way.

Reply:
But why would we ever need to limit who can get the CA?

Reply:
I mean to say *configmaps* that are annotated, not namespaces, so it is badly worded